packages/dataviews/src/components/dataviews/index.tsx

```diff
@@ -167,7 +170,7 @@ export default function DataViews< Item >( {
 					</HStack>
 				</HStack>
 				{ isShowingFilter && <DataViewsFilters /> }
-				<DataViewsLayout />
+				{ containerWidth === undefined ? '' : <DataViewsLayout /> }
```

Comment:
Idea: We should delay rendering `<DataViewsLayout/>` until we have the containerWidth. Currently, this is causing an extra re-render of the table on load and the table is the most resource-intensive.

In my tests, this saved on average `~15%` from the initial load time.

** A couple notes:**
- Since the filters (they render immediately) and the table are rendered separately, there is a moment where the filters come in and then table rows load. 
- We could use a loader instead
- We could delay rendering above the provider. They have the same rough effect on performance.